src/lib/middleware/pat-middleware.ts

```diff
@@ -0,0 +1,43 @@
+import { IUnleashConfig } from '../types';
+import { IAuthRequest } from '../routes/unleash-types';
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+const patMiddleware = (
+    {
+        getLogger,
+        authentication,
+        flagResolver,
+    }: Pick<IUnleashConfig, 'getLogger' | 'authentication' | 'flagResolver'>,
+    { userService }: any,
+): any => {
+    const logger = getLogger('/middleware/pat-middleware.ts');
+    logger.debug('Enabling PAT middleware');
+
+    if (
+        !authentication.enablePersonalAccessToken ||
```

Comment:
Since we are using `flagresolver` now, do we need the extra flag `authentication.enablePersonalAccessToken`?